MPChartLib/src/com/github/mikephil/charting/charts/RadarChart.java

```diff
@@ -358,10 +358,7 @@ public void setDrawWeb(boolean enabled) {
      */
     public void setSkipWebLineCount(int count) {
 
-        if (count < 0)
-            count = 0;
-
-        mWebModulus = count + 1;
+        mSkipWebLineCount = Math.max(0, count);
```

Comment:
Can't that result in 0?
if someone sets the skipWebLineCount to 0 - then the loop that skips weblines has += 0